Fails on ARM with:
```
[8/33] Building CXX object UnitTests/CMakeFiles/UnitTests.dir/Src/UnitTests.cpp.o
FAILED: UnitTests/CMakeFiles/UnitTests.dir/Src/UnitTests.cpp.o
/nix/store/ay4xxbjafns12mn2ilhz4y7d3m3ziwqg-gcc-wrapper-13.3.0/bin/g++ -DCONFIG_RELEASE -DPLATFORM_LINUX -I/build/source/UnitTests/Inc -I/build/source/Modules/Foundation/Inc -I/build/source/Modules/Math/Inc -I/bu>
In file included from /build/source/UnitTests/Src/UnitTests.cpp:20:
/build/source/UnitTests/Src/UnitTests.cpp: In function 'int main(int, char**)':
/build/source/UnitTests/Src/UnitTests.cpp:149:15: error: 'Rule' is not a member of 'tUnitTest'
  149 |         tTest(Rule);
      |               ^~~~
/build/source/UnitTests/Src/UnitTests.h:48:146: note: in definition of macro 'tTest'
   48 | #define tTest(name)  { tUnitTest::rPrintf("\nTesting " #name "\n"); tUnitTest::UnitRequirementNumber = tUnitTest::UnitGoalNumber = 0; tUnitTest::name(); }
      |                                                                                                                                                  ^~~~
```

--- a/pkgs/by-name/ta/tacent/package.nix
+++ b/pkgs/by-name/ta/tacent/package.nix
@@ -33,5 +33,9 @@ stdenv.mkDerivation (finalAttrs: {
     license = lib.licenses.isc;
     maintainers = with lib.maintainers; [ PopeRigby ];
     platforms = lib.platforms.linux;
+    badPlatforms = [
+      # /build/source/UnitTests/Src/UnitTests.cpp:149:15: error: 'Rule' is not a member of 'tUnitTest'
+      "aarch64-linux"
+    ];
   };
 })

